admin/import.php

```diff
@@ -187,8 +207,8 @@
 
     foreach ($page_photos as &$photo)
     {
-      $photo['thumb'] = $flickr->buildPhotoURL($photo, "thumbnail");
-      $photo['src'] = $flickr->get_biggest_size($photo['id'], "medium_800");
+      $photo['thumb'] = $photo['url_t'];
+      $photo['src'] = $photo['url_m'];
```

Comment:
It is important to try to get the biggest size available like  I did not just the medium.

Reply:
This `src` is only used for the 'zoom' function when previewing the photos that are to be imported. It doesn't determine what's actually imported (that's in [ws_functions.php](https://github.com/mistic100/Flickr2Piwigo/pull/7/files#diff-3e3297a9d9faa54640a8e0d574c5ae5cR57)).

The reason I switched it to these image URLs is that they can be retrieved in the `photosets.getPhotos` Flickr API call, and so save on a request.

Reply:
Ok I forgot that